tests/unittest.cpp

```diff
@@ -1124,22 +1124,38 @@ TEST_CASE("json::wvalue::wvalue(std::int64_t)")
 
 TEST_CASE("json::wvalue::wvalue(float)")
 {
-    float f = 4.2;
-    json::wvalue value = f;
+    {
+        float f = 3.14159265359;
+        json::wvalue value = f;
 
-    CHECK(value.t() == json::type::Number);
-    CHECK(value.dump() == "4.2");
+        CHECK(value.t() == json::type::Number);
+        CHECK(value.dump() == "3.14159");
```

Comment:
this is only true for CROW_JSON_FLOAT_PRECISION = 6 , so shouldn't this be part of test equation?